apps/client/src/features/rundown/time-input-flow/TimeInputFlow.module.scss

```diff
@@ -3,15 +3,18 @@
   z-index: 100;
   font-size: 0.75rem;
   font-weight: 600;
-  top: 2px;
-  right: 4px;
+  top: -5px;
+  right: -1px;
   color: inherit;
+  margin: 0.1rem;
 }
 
 .timeAction {
   opacity: 0.4;
   cursor: pointer;
-  padding-right: 0.5em;
+  margin-right: 0.5em;
+  background: transparent;
+  color: white;
```

Comment:
```suggestion
  color: $ui-white;
```

Reply:
Resolved in https://github.com/cpvalente/ontime/pull/1438/commits/d06d5bf80ff52a12ad862541d82a77da51f4c856